README.md

```diff
@@ -6,6 +6,38 @@ OpenTelemetry Google Cloud Trace Exporter allows the user to send collected trac
 
 [Google Cloud Trace](https://cloud.google.com/trace) is a distributed tracing system. It helps gather timing data needed to troubleshoot latency problems in microservice architectures. It manages both the collection and lookup of this data.
 
+### Getting Started
+
+This exporter package assumes your application is [already instrumented](https://github.com/open-telemetry/opentelemetry-js/blob/master/getting-started/README.md) with the OpenTelemetry SDK. Once you are ready to export OpenTelemetry data, you can add this exporter to your application:
+
+sh
```

Comment:
please include opening and closing backticks(```)

Reply:
any update?

Reply:
Done.